WalletWasabi.Fluent/Controls/AnimatedButton.axaml

```diff
@@ -0,0 +1,79 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:behaviors="using:WalletWasabi.Fluent.Behaviors"
+        xmlns:i="using:Avalonia.Xaml.Interactivity"
+        xmlns:ia="using:Avalonia.Xaml.Interactions.Core"
+        xmlns:c="using:WalletWasabi.Fluent.Controls">
+  <Design.PreviewWith>
+    <c:AnimatedButton />
+  </Design.PreviewWith>
+  <Style Selector="Button > Panel > PathIcon">
+    <Setter Property="Opacity" Value="0.5" />
+  </Style>
+    <Style Selector="Button:pointerover > Panel > PathIcon">
+      <Setter Property="Opacity" Value="0.75" />
+    </Style>
+    <Style Selector="PathIcon.hidden">
+      <Setter Property="IsVisible" Value="False" />
+    </Style>
+    <Style Selector="PathIcon.hideAndShow">
+      <Style.Animations>
+        <Animation Duration="0:0:2.0">
```

Comment:
Please use an Easing for this, it's recommended to use `Easing="{StaticResource FluentEasing}"` for all non linear animations.

Reply:
Actually it'd be better if you just use `DoubleTransitions` for opacity property instead of keyframes so that you dont have to declare this much boilerplate :)